test-utils/src/main/java/io/littlehorse/test/LHWorkflowEvent.java

```diff
@@ -0,0 +1,10 @@
+package io.littlehorse.test;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface LHWorkflowEvent {}
```

Comment:
I don't agree with this interface. we should include the definition at LHTest level

Reply:
The problem with that is that the WorkflowEventDef has both a `name` and a `type`. Is that possible to do at the event level? I don't think so...

We could have a `METHOD` level annotation (actually, my first implementation tried to do that) but it requires changing a lot more code in the `TestContext` class.